packages/components/inputs/date-time-input/src/date-time-input.readme.mdx

```diff
@@ -0,0 +1,6 @@
+import { Meta, Markdown } from '@storybook/blocks';
```

Comment:
The `props` page for `DateTimeInput` is missing most of the props
![Screenshot 2024-08-19 at 10 54 52 AM](https://github.com/user-attachments/assets/22760c97-0947-4756-ae10-02166ae06b24)

Reply:
This happened before, sb has issues parsing the types from class-based components. I [added a wrapper-component](https://github.com/commercetools/ui-kit/pull/2881/commits/e943fef6605e26e0738a9136e9c60105a759ae61) for storybook. Alternatively one could have manually added each prop-control via the `argTypes` config of the story, but then you'd have to remember keeping those in sync when updating the component.